### DISCUSSION TITLE
Playlists based on scheduled time

### DISCUSSION THREAD

**Discussion by Vozka1** [Geektoolkit/Dynaframe3#202] [Ideas]
Hi, 
I would like to bring new idea of folder/playlist playback dased on current time.
Idea is that I will have different folders and based on setup and current time will be played adequate folder to that time.

f.e.
I have 3 folders:
/kids - will contain fairytale pictures
/family - will contain some family pictures
/other - f.e. could contain some different pictures which I like ;)

/kids will be played from 7:00 to 12:00
/family will be played from 12:00 to 22:00
/other will be played 22:00 to 24:00

Thank you
Vozka

**Comment by Salutatous-22** [comment]
Hi
I found you have a good suggestion
and if is possible to add at this suggestion a period where the sow are stop and the monitor are turn off
why run the show when nobody are there 

regards
Salutatous (Chris)

**Comment by garethsnaim** [comment]
I would like to add to this. My company would love for the ability to show images at specific times on specific days of the week. We are in catering and want to show certain menus at certain times on certain days. I dont see that in any other pi software.
